src/main/java/com/nvs/store/models/product/Product.java

```diff
@@ -10,7 +10,7 @@
 
 @Data
 @Builder
-@NoArgsConstructor
+@NoArgsConstructor(force = true)
```

Comment:
why

Reply:
this is because fields are annotated with "@NonNull"